### PR TITLE
Bump BEEFY

### PR DESCRIPTION
**Pull request by andresilva**
Replacement for https://github.com/paritytech/polkadot/pull/2905.

### CODE REVIEW

**Review by ordian — APPROVED** [review]

**Review by adoerr — APPROVED** [review]
LGTM

**Comment by andresilva** [comment]
Don't merge this yet, I will bump again after https://github.com/paritytech/grandpa-bridge-gadget/pull/156.

**Comment by andresilva** [comment]
bot merge

**Pull request merged**
